src/sora-academy.md

```diff
@@ -1,3 +1,53 @@
 # SORA Academy
 
-The [SORAcademy](https://medium.com/sora-academy) provides tutorial on how to use SORA.
+Thanks to blockchain technology, finance has become more inclusive and
+less centralized, sparking a sector known as Decentralized Finance,
+and with access to a crypto asset wallet, people can participate in
+activities such as swapping tokens (exchanges), liquidity provision,
+NFT Markets, and more. The SORA Academy is the best way to get you started in DeFi on the SORA network
```

Comment:
```suggestion
NFT Markets, and more. The SORA Academy is the best way to get you started in DeFi on the SORA network.
```

Reply:
@WRRicht3r you didn't apply this change

Reply:
and a couple of others as well. I've noticed you applied them and then overwritten the changes